From 4aeb89c83e0a0dd15bcedd7a9358eb1d2c470270 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 29 Jan 2015 09:26:30 +0100
Subject: [PATCH 01/42] build: set iridium version

---
 chrome/VERSION | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index c9285b7..6994a5b 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=41
 MINOR=0
-BUILD=2272
-PATCH=104
+BUILD=0
+PATCH=0
-- 
2.4.0

