From 69ae251a07153e697cef5d78490d6a558926b7fb Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 13 Oct 2014 12:28:51 +0200
Subject: [PATCH 17/17] Extend our User-Agent string

---
 content/common/user_agent.cc | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index 2255bad..8165ae6 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -181,16 +181,21 @@ std::string BuildUserAgentFromOSAndProduct(const std::string& os_info,
   // Derived from Safari's UA string.
   // This is done to expose our product name in a manner that is maximally
   // compatible with Safari, we hope!!
+  chrome::VersionInfo vi;
+  std::string ua2;
+  if (!vi.is_valid())
+  	ua2 = "IridiumCRV/" + vi.Version();
   std::string user_agent;
   base::StringAppendF(
       &user_agent,
-      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) %s Safari/%d.%d",
+      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) %s Safari/%d.%d %s",
       os_info.c_str(),
       WEBKIT_VERSION_MAJOR,
       WEBKIT_VERSION_MINOR,
       product.c_str(),
       WEBKIT_VERSION_MAJOR,
-      WEBKIT_VERSION_MINOR);
+      WEBKIT_VERSION_MINOR,
+      ua2);
   return user_agent;
 }
 
-- 
2.4.0

