From a37798ec2601d46658389f8f884a0c7a5fbe4b25 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 2 Apr 2015 10:13:30 +0200
Subject: [PATCH 48/48] iridium-browser 41.1

---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index 6994a5b..fdf56ff 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=41
-MINOR=0
+MINOR=1
 BUILD=0
 PATCH=0
-- 
2.4.0

