From c8fb2f6e8603ec791a280b41d44c8dfacbe79f4e Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 2 Mar 2015 02:04:17 +0100
Subject: [PATCH 42/42] iridium-browser 40.2

---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index 445d084..e576378 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=40
-MINOR=1
+MINOR=2
 BUILD=0
 PATCH=0
-- 
2.4.0

