From 2053be6156f9c4a954cd3a3a5f3c41017c7f6b36 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 29 Jan 2015 10:39:51 +0100
Subject: [PATCH 37/40] openSUSE patch chromium-sandbox-pie.patch

---
 sandbox/linux/sandbox_linux.gypi | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/sandbox/linux/sandbox_linux.gypi b/sandbox/linux/sandbox_linux.gypi
index ba530c0..72875be 100644
--- a/sandbox/linux/sandbox_linux.gypi
+++ b/sandbox/linux/sandbox_linux.gypi
@@ -204,7 +204,10 @@
       ],
       'cflags': [
         # For ULLONG_MAX
-        '-std=gnu99',
+        '-std=gnu99 -fPIE',
+      ],
+      'ldflags': [
+        '-pie',
       ],
       'include_dirs': [
         '../..',
-- 
2.4.0

