From 16847898260e62de1605d002efca96fb8f677486 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 29 Jan 2015 10:39:51 +0100
Subject: [PATCH 37/42] openSUSE patch chromium-sandbox-pie.patch

---
 sandbox/linux/sandbox_linux.gypi | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/sandbox/linux/sandbox_linux.gypi b/sandbox/linux/sandbox_linux.gypi
index 89c830e..8412de8 100644
--- a/sandbox/linux/sandbox_linux.gypi
+++ b/sandbox/linux/sandbox_linux.gypi
@@ -203,7 +203,10 @@
       ],
       'cflags': [
         # For ULLONG_MAX
-        '-std=gnu99',
+        '-std=gnu99 -fPIE',
+      ],
+      'ldflags': [
+        '-pie',
       ],
       'include_dirs': [
         '../..',
-- 
2.4.0

